cylc/flow/cfgspec/globalcfg.py

```diff
@@ -1399,6 +1399,17 @@ def default_for(
                  desc=f'''
                 {LOG_RETR_SETTINGS['retrieve job logs command']}
 
+                .. note::
+                   The default command (``rsync -a``) means that the retrieved
+                   files (and the directories above including ``job/log``) get
+                   the same permissions as on the remote host. This can cause
+                   problems if the remote host uses different permissions to the
+                   scheduler host (e.g. no world read access). To avoid this
+                   problem you can set the command to
```

Comment:
Exceeded max line length.

```suggestion
                   problems if the remote host uses different permissions to
                   the scheduler host (e.g. no world read access). To avoid
                   this problem you can set the command to
```